.dir-locals.el

```diff
@@ -17,6 +17,7 @@
                  (python-tab-width . 4)
```

Comment:
First time I've seen us making changes to these files....or that I've seen as part of a project period (that I can recall, I don't remember having any `*.el` files in previous projects). They seem to be config-esque type files. Is that assumption correct? What are they for?

Reply:
These have been in for a while. They're basically the Emacs equivalent to the Gitpod configs.